action.yaml

```diff
@@ -54,6 +54,46 @@ runs:
 
         fi
 
+    - name: Test Report
+      uses: dorny/test-reporter@v1
+      id : test-result
+      if: ${{ env.cmd_type == 'UNIT_TEST' &&  env.test_report_path !='' && env.reporter !='' }}                   
+      with:
+        name: Unit Test                          
+        path: ${{ env.test_report_path}}              # '**/target/**/*.xml'   
+        reporter: ${{ env.reporter }}                 # Format of test results. 
+        only-summary: 'true' 
+        fail-on-error: ${{ env.fail-on-error || false }}     # Set action as failed if test report contains any failed test
+
+    - name: Copy test report to target folder
+      if: ${{ always() && env.copy-to-target-path !='' }}
```

Comment:
at least `env.cmd_type == 'UNIT_TEST'` needs to be here also. IF someone just add the `copy-to-target-path` env variable while building - for instance - this step gets triggered also. We need to tie this to unit tests step only.

Reply:
Updated condition for `'Copy test report to target folder'` step with `env.cmd_type == 'UNIT_TEST'`.